core/cas-server-core-services/src/main/java/org/apereo/cas/services/DefaultRegisteredServiceAccessStrategy.java

```diff
@@ -412,4 +384,13 @@ public String toString() {
                 .append("rejectedAttributes", this.rejectedAttributes)
                 .toString();
     }
+
+    private static Set<String> objectToSetOfStrings(final Object objVal) {
```

Comment:
Move this to CollectionUtils. In fact, there may be something similar there already?